src/main/java/Elements/Person.java

```diff
@@ -0,0 +1,67 @@
+package Elements;
+
+public class Person {
+    private int id;
+    private String name;
+    private String lastName;
+    private String phone;
+    private String phoneAlternative;
+    private String email;
+    private String type;
+
+    public int getId() {
+        return id;
```

Comment:
en los getters no debería ser `return this.id`?

Reply:
aquí no es cambio, es duda.

Reply:
de eso si no estoy seguro